create-list.md

```diff
@@ -4,6 +4,6 @@
 - Wait at least a couple of weeks after creating a list before submitting to this list to give it a chance to mature.
 - Search this list before making a new one, as yours may be a duplicate. If it is, try and contribute to the best one instead of making your own.
 - A link back to this list from yours, so users can discover more lists, would be appreciated. 
-- Make sure you read the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) first.
+- Make sure you read the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) before submitting a PR to this list to add your own list.
```

Comment:
I suggest something like:

> ...before submitting a request for your list to be added here.

The whole `to this list to add your list` thingy sounds a bit confusing to me.